canonical-web-and-design/maas-ui
# Build zones list form

## Done

- Add form to header to add zone to list
- Add onSuccess prop to close form when zone is added
- Add tests for form 
- We need some help with testing - we want to test that the form only hides when a zone has been added successfully but haven't managed to do it

## QA

- run `yarn start` and go to http://0.0.0.0:8400/MAAS/r/zones
- Check when a duplicate zone is attempted to be added the form remains open, however when a zone is successfully added the form closes

![Screenshot 2021-06-09 at 15 03 41](https://user-images.githubusercontent.com/58959073/121369601-e8222900-c933-11eb-9392-2343e6e63be5.png)
